gapic/src/main/com/google/gapid/models/CommandStream.java

```diff
@@ -376,6 +377,16 @@ public CommandIndex getIndex() {
           getPath(Path.CommandTreeNode.newBuilder()).build());
     }
 
+    public String getIndexString() {
+      if (data == null) {
+        return "";
```

Comment:
Is it safe to return an empty string? Shouldn't we return null here (similar to function above)?

Reply:
yeah it would be safe, usually for string in java an empty one would be safer than a null.